### DISCUSSION TITLE
Cannot edit media

### DISCUSSION THREAD

**Discussion by autocreation** [awcodes/filament-curator#348] [Q&A]
I am running the latest 3.1.1 version (just upgraded) and have added the themes, cropperjs and hoped anything else needed.  I have no issues creating a new item that makes use of the CuratorPicker, can upload new images, attach to my model,...  

However, if I then go to edit one of these items the image is displayed properly with filename / size at the bottom of the image, but there is no way for me to edit this and potentially replace the image.  I was almost 99% certain this worked in 2.x but it has been a long time since I worked on this project.  

The picker is defined as follows:
```
                CuratorPicker::make('media_id')
                    ->required()
                    ->disk('public')
                    ->directory('images/events')
                    ->limitToDirectory()
                    ->preserveFilenames()
                    ->imageResizeTargetHeight('288')
                    ->imageResizeTargetWidth('518'),

```

What might I be missing?  Thanks in advance!

**Comment by awcodes** [comment]
You can only replace the image from the Edit page. You can not do it from the picker modal.

**Reply by autocreation** [reply]
That picker code is from the form function and used by both the create and edit options. The create works fine, but the edit allows no changes at all, cannot be clicked or replaced.

**Reply by awcodes** [reply]
If the field is only set to allow one image then you have to remove it before you can add another in it's place.

![Screenshot 2023-11-21 at 8 39 36 AM](https://github.com/awcodes/filament-curator/assets/3596800/6bad895b-e520-48c1-8ee8-ab0734d2789f)

**Comment by autocreation** [comment]
Interesting but my image does not have the dot menu.  It does show the filename / size.  Only one image is allowed.  Thanks for the assistance!

**Comment by DonatasIk** [comment]
I did not fully understood the reason why the dot menu is not appearing, as I have the same issue myself. Currently, I have the latest version installed (v3.6.2) and would like to replace the current image from the Edit page of the Resource. Unfortunatelly, it only shows the filename / size on the bottom of my my image. How to make the dot menu appear? Do I need to set some value in my config?

My picker is defined like this:
```
CuratorPicker::make('media_id')
        ->label('Some label')
```

**Reply by awcodes** [reply]
Not sure what to tell you, sorry. My best guess is something is off with your styles.
![Screenshot 2024-07-22 at 10 17 11 AM](https://github.com/user-attachments/assets/c0d5bc41-5871-4b0f-bbb1-44ce07a299de)

**Comment by autocreation** [comment]
Actually I found the three dots to be in the middle of my images and VERY light.  Maybe the same for you?  Try running the mouse over the top of the photo from edge to edge and see if the pointer changes - might help.

**Reply by DonatasIk** [reply]
Indeed, I also see three very light coloured dots in the middle top of my images. Seems like it's something associated with the styles then. I wonder if it's because some other plugin is installed

**Comment by awcodes** [comment]
Make sure you follow the installation instructions. especially the part about setting up a custom theme with filament.

**Comment by tonybyng** [comment]
I've got mine in the middle at the top as well.  Three light dots with no dark background to highlight them. Was fairly sure I'd got everything right css wise. No other issues anywhere else anyway. Will have to go through everything again and make sure I didnt miss anything. Glad to have found this entry though as at least I'm not the only one !

**Comment by DonatasIk** [comment]
Okay, so I figure it out, finally. I was living with this issue for some time now, but I randomly came to this again and now I see where I made my initial mistake while installing the package and setting up the styles. Basically I included package views (`'./vendor/awcodes/filament-curator/resources/**/*.blade.php',`) inside the incorrect `tailwind.config.js` config file. 

The view should be included inside Filament's custom `tailwind.config.js` config, which is located in `resources/css/filament/admin` directory and not the root one.

```js
// Correct path: resources/css/filament/admin/tailwind.config.js
content: [
    './vendor/awcodes/filament-curator/resources/**/*.blade.php',
    // other paths...
]
